Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -19,17 +21,15 @@ namespace HoloToolkit.Unity.InputModule
     /// Edit -> Project Settings -> Player -> Settings for Windows Store -> Publishing Settings -> Capabilities
     /// or in your Visual Studio Package.appxmanifest capabilities.
     /// </summary>
-    public partial class KeywordManager : MonoBehaviour
+    public partial class KeywordManager : BaseInputSource
```

Comment:
Can we create a new class (SpeechInputSource) instead of modifying the existing Keyword manager? Then, if it works well, we could simply eventually delete the old KeywordManager.